.github/workflows/build-and-push.yml

```
@@ -19,22 +19,39 @@ jobs:
             quayio_username: "QUAY_IMAGE_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_BUILDER_TOKEN"
             tag: "centos7"
+            suffix: "centos7"
           - dockerfile_path: "1.20"
             dockerfile: "Dockerfile"
             registry_namespace: "centos7"
             quayio_username: "QUAY_IMAGE_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_BUILDER_TOKEN"
             tag: "centos7"
+            suffix: "centos7"
           - dockerfile_path: "1.20"
             dockerfile: "Dockerfile.c9s"
             registry_namespace: "sclorg"
             quayio_username: "QUAY_IMAGE_SCLORG_BUILDER_USERNAME"
             quayio_token: "QUAY_IMAGE_SCLORG_BUILDER_TOKEN"
             tag: "c9s"
+            suffix: "c8s"
```

```suggestion
            suffix: "c9s"
```
But I would rather won't state suffix here at all, as it seems, that always stays that suffix=tag.